metadata/gc.go

```diff
@@ -680,6 +768,21 @@ func isRootRef(bkt *bolt.Bucket) bool {
 	return false
 }
 
+func isExpired(ctx context.Context, k []byte, bkt *bolt.Bucket, expTheshold time.Time) bool {
```

Comment:
suggest to use `isExpiredImage` since it's only used by image.

Reply:
Updated